src/object_schema.cpp

```diff
@@ -63,27 +63,27 @@ ObjectSchema::ObjectSchema(std::string name, IsEmbedded is_embedded, std::initia
     }
 }
 
-PropertyType ObjectSchema::from_core_type(Table const& table, ColKey col)
+PropertyType ObjectSchema::from_core_type(ColKey col)
 {
     auto flags = PropertyType::Required;
-    auto attr = table.get_column_attr(col);
+    auto attr = col.get_attrs();
     if (attr.test(col_attr_Nullable))
         flags |= PropertyType::Nullable;
     if (attr.test(col_attr_List))
         flags |= PropertyType::Array;
-    switch (table.get_column_type(col)) {
-        case type_Int:       return PropertyType::Int | flags;
-        case type_Float:     return PropertyType::Float | flags;
-        case type_Double:    return PropertyType::Double | flags;
-        case type_Bool:      return PropertyType::Bool | flags;
-        case type_String:    return PropertyType::String | flags;
-        case type_Binary:    return PropertyType::Data | flags;
-        case type_Timestamp: return PropertyType::Date | flags;
-        case type_OldMixed:  return PropertyType::Any | flags;
-        case type_ObjectId:  return PropertyType::ObjectId | flags;
-        case type_Decimal:   return PropertyType::Decimal | flags;
-        case type_Link:      return PropertyType::Object | PropertyType::Nullable;
-        case type_LinkList:  return PropertyType::Object | PropertyType::Array;
+    switch (col.get_type()) {
+        case col_type_Int:       return PropertyType::Int | flags;
+        case col_type_Float:     return PropertyType::Float | flags;
+        case col_type_Double:    return PropertyType::Double | flags;
+        case col_type_Bool:      return PropertyType::Bool | flags;
+        case col_type_String:    return PropertyType::String | flags;
+        case col_type_Binary:    return PropertyType::Data | flags;
+        case col_type_Timestamp: return PropertyType::Date | flags;
+        case col_type_OldMixed:  return PropertyType::Any | flags;
+        case col_type_ObjectId:  return PropertyType::ObjectId | flags;
+        case col_type_Decimal:   return PropertyType::Decimal | flags;
+        case col_type_Link:      return PropertyType::Object | PropertyType::Nullable;
+        case col_type_LinkList:  return PropertyType::Object | PropertyType::Array;
```

Comment:
Potentially we could now add a case for converting `col_type_BackLink` into a `PropertyType::LinkingObjects` but that's not really part of this PR (not blocking).

Reply:
We also have `col_type_OldStringEnum` but that should not be possible to reach since AFAIK if we had any enum columns they should have all been converted to string columns in the migration to core 6.